README.md

```diff
@@ -2079,39 +2078,43 @@ to taxes on equity compensation:
-    of shares you own.
-  - **Vesting and liquidity**: When you will actually receive the shares and when you’ll be able
-    to sell them.
-  - **Tax**: Both the purchase and the sale of stock can require that you pay taxes—sometimes
-    very large amounts.
-    There are several kinds of taxes that may apply when you have stock:
-    income, capital gains, and AMT.
-- 🔹 Knowing how many shares of stock or stock options you have is meaningless unless you
-  know the number of outstanding shares.
-  What matters is the percentage ownership (typically in basis points) of the company the
-  shares represent, including the [details](#counting-shares) of how the total is counted.
-  Some companies don’t volunteer this information unless you specifically ask for it, but
-  it’s always fair to do so, because without it the offer of shares is almost meaningless.
-- 🔹 It is critical to understand dilution and [risk](#stages-of-a-startup) to ascertain the
-  possible future value of equity for the company and its stage.
-  [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets provides some
-  additional thoughts.
+In summary, you have to determine or make educated guesses about several things:
+
+- **Equity value**: Determined by the value the company may have in the future, and the number
+  of shares you may own?
+  - **Percentage ownership**: As we’ve mentioned, knowing how many shares of stock or stock
+    options you have is meaningless unless you know the number of outstanding shares.
+    What matters is the percentage ownership of the company the shares represent, including
+    the [details](#counting-shares) of how the total is counted.
+  - **Risk**: It is critical to understand [risk](#stages-of-a-startup) in the business and dilution
+    to ascertain the possible future value of equity for the company and its stage.
+    [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets provides some
+    additional thoughts.
+- **Vesting**: When you will receive the equity, as well as whether you’re able to exercise
+  stock options (and pay the associated costs and taxes).
+  And whether you can do all this before your exercise window expires.
+- **Liquidity**: When you’ll be able to sell the shares, and if that is likely to be for a
+  profit at that time.
+  (We talk about liquidity of private stock next.)
+- **Tax**: The tax implications of your possible grant, exercise, vesting, and sale, in terms
+  of ordinary income tax, employment tax, long-term capital gains, and alternative minimum
+  tax.
+
+That’s a lot, and even so, decisions are uncertain, but it is possible to make much more
+informed decisions once you have this information.
 
 ### What is private stock worth?
 
-If you have equity in a private company, how can you think about its value?
+We now turn to the question of determining the value of private company stock.
```

Comment:
This sounds like we're going to answer the question. "This is how!" You cannot determine it, so "determine" is the wrong word. The next line is actually fine on its own, given the title of the section, although the line is poorly phrased. First line: "Because stock in private companies usually can't be sold, there is no consensus on its value, which can only be estimated."

Reply:
I'm fine with deleting this whole sentence if you think that's better!